### PR TITLE
feat: added cart page

### PR DESCRIPTION
**Pull request by sks-21**
**Added cart page**
Includes the product and the price details

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Looks good to me

File: cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.page-container {
+    display: grid;
+    height: 100vh;
+    grid-template-rows: 8rem 1fr;
+    grid-template-areas:
+        "nav nav"
+        "main main"
+    ;
+    gap: 20px;
```
Comment (fahd23) [inline cart/cart.css]: If possible convert px to rem

File: home/card.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+.horizontal-card-content, .horizontal-card-img-container,.card-details {
+    height: 26rem;
+}
+
+.card-quantity {
+    margin-top: 8px;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 5px;
+    /* height:fit-content; */
```
Comment (fahd23) [inline home/card.css]: If not needed remove commented code

File: home/card.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+
+.quantity-btn {
+    font-size: 2rem;
+    /* width: 0.4rem; */
```
Comment (fahd23) [inline home/card.css]: same here

**Review by fahd23 — APPROVED** [review]
Given some suggestions, looks good to merge

**Pull request merged**